gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjureBasePlugin.java

```diff
@@ -105,8 +108,7 @@ private static void createOutgoingConfiguration(Project project, TaskProvider<Co
             conf.setCanBeResolved(false);
             conf.setCanBeConsumed(true);
             conf.setVisible(true);
-            conf.getAttributes()
-                    .attribute(Usage.USAGE_ATTRIBUTE, project.getObjects().named(Usage.class, "conjure"));
+            conf.getAttributes().attribute(CONJURE_USAGE, project.getObjects().named(Usage.class, "conjure"));
```

Comment:
Note that if you specify a specific attribute on the consumer side, gradle considers both configurations that match it, but also configurations that don't have that attribute defined at all.
Therefore, if there are multiple published configurations (i.e. having attributes), some specifying this attribute and others not specifying it, gradle will say they are ambiguous on the consumer side, unless you also specify a disambiguation rule.

I think the previous thing made sense because:
1. a different usage makes it impossible to depend on both the java variant and the conjure variant _in the same configuration_ (which is probably what you want)
2. it doesn't run into the problem above